client/state/sites/plans/actions.js

```diff
@@ -122,7 +121,7 @@ export function fetchSitePlans( siteId ) {
  * @returns {Object} the corresponding action object
  */
 export function fetchSitePlansCompleted( siteId, data ) {
-	data = reject( data, '_headers' );
+	delete data._headers;
```

Comment:
Shouldn't we stick to a functional programming style for this global state tree actions?